systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMaker2ST.java

```diff
@@ -1069,6 +1077,453 @@ void testRestoreOffsetsInConsumerGroup(ExtensionContext extensionContext) {
         assertDoesNotThrow(() -> ClientUtils.waitForClientTimeout(sourceConsumerName, namespaceName, 1));
     }
 
+    /**
+     * Test mirroring messages by MirrorMaker 2.0 over tls transport using scram-sha-512 auth
+     * while user Scram passwords, CA cluster and clients certificates are changed.
+     */
+    @ParallelNamespaceTest
+    @SuppressWarnings({"checkstyle:MethodLength"})
+    void testKMM2RollAfterSecretsCertsUpdateScramsha(ExtensionContext extensionContext) {
+        TestStorage testStorage = new TestStorage(extensionContext);
+
+        String kafkaClusterSourceName = testStorage.getClusterName() + "-source";
+        String kafkaClusterTargetName = testStorage.getClusterName() + "-target";
+        String topicSourceNameA = MIRRORMAKER2_TOPIC_NAME + "-a-" + rng.nextInt(Integer.MAX_VALUE);
+        String topicSourceNameB = MIRRORMAKER2_TOPIC_NAME + "-b-" + rng.nextInt(Integer.MAX_VALUE);
+        String topicTargetNameA = kafkaClusterSourceName + "." + topicSourceNameA;
+        String topicTargetNameB = kafkaClusterSourceName + "." + topicSourceNameB;
+
+        String kafkaUserSourceName = testStorage.getClusterName() + "-my-user-source";
+        String kafkaUserTargetName = testStorage.getClusterName() + "-my-user-target";
+        String kafkaTlsScramListenerName = "tlsscram";
+
+        // Deploy source kafka with tls listener and SCRAM-SHA authentication
+        resourceManager.createResource(extensionContext, KafkaTemplates.kafkaPersistent(kafkaClusterSourceName, 1, 1)
+            .editSpec()
+                .editKafka()
+                .addToConfig("min.insync.replicas", 1)
```

Comment:
This is by default in `kafkaPersistent -> defaultKafka` builder set, so you don't need to set it here too.